doc/reference/tooling/tt_cli/export.rst

```diff
@@ -0,0 +1,145 @@
+.. _tt-export:
+
+Exporting data
+==============
+
+..  admonition:: Enterprise Edition
+    :class: fact
+
+    This command is supported by the `Enterprise Edition <https://www.tarantool.io/compare/>`_ only.
+
+
+..  code-block:: console
+
+    $ tt [crud] export URI FILE SPACE [EXPORT_OPTION ...]
+
+``tt [crud] export`` exports a space's data to a file.
+The ``crud`` command is optional and can be used to export a cluster's data by using the `CRUD <https://github.com/tarantool/crud>`_ module. Without ``crud``, data is exported using the :ref:`box.space <box_space>` API.
+
+``tt [crud] export`` takes the following arguments:
+
+*   ``URI``: The URI of a router instance if ``crud`` is used. Otherwise, it should specify the URI of a storage.
+*   ``FILE``: The name of a file for storing exported data.
+*   ``SPACE``: The name of a space from which data is exported.
+
+..  NOTE::
+
+    You should have :ref:`read access <authentication-owners_privileges>` to the required space to be able to export data.
```

Comment:
It's unclear who is `you` in this sentence. The reader runs `tt` outside of Tarantool.
Shorter wording: `to the required space to be able to export data.` > `to the space to export its data`.

Reply:
Will update to `Read access to the space is required to export its data.`